src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -116,10 +116,7 @@ class PolicyManager final : public Singleton<PolicyManager>
             newPolicy["indexer"]["ssl"]["key"] = "";
         }
         newPolicy["indexer"]["name"] =
-            STATES_VD_INDEX_NAME_PREFIX +
-            (newPolicy.at("vulnerability-detection").contains("clusterName")
-                 ? newPolicy.at("vulnerability-detection").at("clusterName").get_ref<const std::string&>()
-                 : "default");
+            STATES_VD_INDEX_NAME_PREFIX + newPolicy.at("clusterName").get_ref<const std::string&>();
```

Comment:
This should be lower-case formatted due to restrictions on the indexer
```json
{
    "index":
    {
        "_index": "wazuh-states-vulnerabilities-Thinkpad-Sebas",
        "_id": "001_ff5f53a5ed3d2659c0f0b5de579951d332ca47b3_CVE-2024-22667",
        "status": 400,
        "error":
        {
            "type": "invalid_index_name_exception",
            "reason": "Invalid index name [wazuh-states-vulnerabilities-Thinkpad-Sebas], must be lowercase",
            "index": "wazuh-states-vulnerabilities-Thinkpad-Sebas",
            "index_uuid": "_na_"
        }
    }
}
```

```suggestion
            STATES_VD_INDEX_NAME_PREFIX + Utils::toLowerCase(newPolicy.at("clusterName").get_ref<const std::string&>());
```

Reply:
After the fix, vulnerabilities are indexed
![image](https://github.com/wazuh/wazuh/assets/34063881/96123095-d5af-471a-99f2-7a88df1ea01b)

Reply:
good catch!